ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -25,25 +72,17 @@ public final class Atomic<Value> {
 	}
 	
 	/// Initializes the variable with the given initial value.
-	public init(_ value: Value) {
+	public init(_ value: Value, mutex: MutexType = PosixThreadMutex()) {
```

Comment:
Picking a particular mutex implementation is exactly what happens in `MutableProperty`